Assets/MirageXR/Common/Scripts/ActivityCard/ActivityCard.cs

```diff
@@ -10,28 +10,28 @@ public class ActivityCard : MonoBehaviour
     {
         // Define all the components.
 
-        private AudioSource _audio;
+        private AudioSource audio;
```

Comment:
I'm not sure I should change the `_privateField` style to `privateField`. Maybe we should take a vote? @Alvareon @BenediktHensen @Callustian @jafarigit @robhillman97

Reply:
I think it is a matter for mutual arrangement. @BenediktHensen, what problem do we want to solve? What's wrong with old name?

Reply:
Let's just vote. 
I vote for the option `_privateField`
@Alvareon @BenediktHensen @Callustian @jafarigit @robhillman97

Reply:
I vote for  `_privateField`